package.json

```diff
@@ -111,9 +119,5 @@
     "vueify": "^9.4.1",
     "watchify": "^3.7.0",
     "yargs": "^8.0.2"
-  },
-  "engines" : { 
-    "node" : ">=16.13.2",
-    "npm": ">=8.1.2"
```

Comment:
@volterra79 be careful, I think there is an oversight here

Reply:
@Raruto aligned with dev. Thanks